src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -58,6 +58,20 @@ static void LogFindResults(ActiveLicenseFindResult result)
 
                 Logger.Info(report.ToString());
             }
+
+#if REGISTRYLICENSESOURCE
+            if (result.Location.StartsWith("HKEY_"))
```

Comment:
The way the license selection code works, yes if a license with the same expiration date is found in file location and the registry, the file location will be chosen as the active license.